docs/persistence.md

```diff
@@ -0,0 +1,147 @@
+---
+description: __noun__ firm or obstinate continuance in a course of action in spite of difficulty or opposition.
+---
+
+# Persistence
+
+Planning the design of StimulusReflex user interfaces is simpler than what we're used to, whether that's submitting a form and loading the results or using a JavaScript SPA framework to communicate with an API. Since your applications won't have a client state, we estimate that 80% of the effort previously required is now behind you. It does, however, require some unlearning and the willingness to rethink how you approach persisting the state of your application. We recognize that this can be jarring at first and that even positive changes can feel like work.
+
+The best way to start thinking about how interfaces are designed in a StimulusReflex world is to remember that when you access a URL in your browser, you are seeing the current state of your UI; there is no mounting process. Your request goes through the Rails router to Action Pack where your controller renders your view templates and sends them down the wire. At this stage, StimulusReflex is not yet involved. You can call ActiveRecord, access data from Redis or the Rails cache, and set instance variables that get picked up in your view. This is Rails in full 2004 glory.
```

Comment:
`s/This is Rails in full 2004 glory./This is Rails in all it's server rendered glory./`